modules/cli-options/src/main/scala/scala/cli/commands/UninstallCompletionsOptions.scala

```diff
@@ -0,0 +1,18 @@
+package scala.cli.commands
+
+import caseapp._
+
+// format: off
+@HelpMessage("Uninstalls completions from your shell")
+final case class UninstallCompletionsOptions(
+  @Recurse
+    logging: LoggingOptions = LoggingOptions(),
+  @Recurse
+    opts: UninstallCompletionsOpts = UninstallCompletionsOpts(),
```

Comment:
`UninstallCompletionsOpts` nested inside of `UninstallCompletionsOptions` is super confusing.
I understand the intention (not to double on `LoggingOptions` in `UninstallOptions`), but still.
a better differentiation by naming + a comment would be good, unless there's a better way of doing this (is there? cc @alexarchambault)
maybe smth like:
 ```suggestion
    opts: StrippedUninstallCompletionsOptions = StrippedUninstallCompletionsOptions(),
```

Reply:
Usually, I add a `Shared` prefix, like `SharedUninstallCompletionsOptions`.

Reply:
I talked with @alexarchambault and he suggested to use `SharedUninstallCompletionsOptions` naming.